conda-recipes/llvmdev/meta.yaml

```diff
@@ -1,8 +1,9 @@
-{% set shortversion = "11.1" %}
-{% set version = "11.1.0" %}
-{% set sha256_llvm = "ce8508e318a01a63d4e8b3090ab2ded3c598a50258cc49e2625b9120d4c03ea5" %}
-{% set sha256_lld = "017a788cbe1ecc4a949abf10755870519086d058a2e99f438829aef24f0c66ce" %}
-{% set build_number = "5" %}
+{% set shortversion = "14.0" %}
```

Comment:
We'll need split build for linux-aarch64 to use llvm11 as discussed in https://github.com/numba/numba/issues/8756.

See 
https://github.com/numba/llvmlite/pull/599/files#diff-a97126465bc884942a676793bbd8f9443dc410f5aa05c57d9bbf1d0ce5a6c437R2-R18 for reference.